src/ui/src/components/ResetDemoButton.tsx

```diff
@@ -0,0 +1,34 @@
+"use client";
+
+import { API_URL } from "@/utils/constants";
+import { Button } from "@trussworks/react-uswds";
+
+const ResetDemoButton: React.FC = () => {
+  const handleResetDemo = async () => {
+    const response = await fetch(`${API_URL}/demo/reset`, {
+      method: "POST",
+      credentials: "include",
+      headers: {
+        "Content-Type": "application/json",
+      },
+    });
+    if (response.ok) {
+      window.location.reload();
+    } else {
+      console.error("Failed to reset demo data");
+    }
+  };
+  return (
+    <Button
+      type="button"
+      outline
+      className="margin-left-2"
+      style={{ whiteSpace: "nowrap" }}
```

Comment:
nit: I think `text-wrap` [class](https://designsystem.digital.gov/utilities/text-styles/#whitespace-formatting) would take care of this styling.